src/sas/qtgui/Plotting/QRangeSlider.py

```diff
@@ -230,6 +241,10 @@ def onRelease(self, ev):
 
     def clear_markers(self):
         """
-        Should be no way to clear the markers
+        Disconnect the input and clear the callbacks
         """
-        pass
+        if self.input:
+            self.input.textChanged.disconnect(self.inputChanged)
```

Comment:
This will get disconnected when the slider is closed/removed but explicit disconnect is always good. We should probably strive at having disconnects for each signal connect.